core/l10n_init.js

```diff
@@ -61,6 +61,8 @@ define('core/l10n_init',
     }
 
     function _transformLocale(locale) {
+        // Use default locale "en-US" if given locale is undefined (BUG 1138558)
+        locale = locale || 'en-US';
```

Comment:
I think the error you are referencing only happens in development since our development server doesn't properly 404 if a locale script is not present. A little bit annoying, but not actually a bug.